dcos-log/api/v2/handlers.go

```diff
@@ -38,75 +38,110 @@ const (
 	eventStreamContentType = "text/event-stream"
 )
 
+type errSetupFilesAPIReader struct {
+	msg  string
+	code int
+}
+
+// Error implements errors interface.
```

Comment:
btw, you don't need to document exported methods for unexported types. the linter lets you get by without it.

Reply:
TIL